common/dex/base/dex-passwords.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: dex-passwords
+data:
+  DEX_USER_PASSWORD: $2y$12$4K/VkmDd1q1Orb3xAt82zu8gk7Ad6ReFR4LCP9UeYE90NLiN9Df72
```

Comment:
This value should be encrypted with base64 or provided as `stringData.DEX_USER_PASSWORD`.

Using this in current form presents this error on `kubectl apply`:
```
Error from server (BadRequest): error when creating "STDIN": Secret in version "v1" cannot be handled as a Secret: illegal base64 data at input byte 52
```